### PR TITLE
Update CHANGELOG.md for patch release

### PR DESCRIPTION
**Pull request by annablender**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## 1.3.1 (May 25, 2021)
+
+FEATURES:
```
Comment (paynejacob) [inline CHANGELOG.md:3]: We can drop the empty sections

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+## 1.3.1 (May 25, 2021)
```
Comment (paynejacob) [inline CHANGELOG.md:1]:
```suggestion
## 1.3.1 (May 25, 2022)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+
+
+
+ENHANCEMENTS:
+
+
+
+BUG FIXES:
+
+* Patch & re-assign `rke_cluster_yaml` post create. See https://github.com/rancher/terraform-provider-rke/pull/327
```
Comment (paynejacob) [inline CHANGELOG.md:13]:
```suggestion
* Patch & re-assign `rke_cluster_yaml` post create. See [#327](https://github.com/rancher/terraform-provider-rke/pull/327)
```

**Review by paynejacob — CHANGES_REQUESTED** [review]

**Review by paynejacob — APPROVED** [review]

**Review by snasovich — APPROVED** [review]

**Comment by snasovich** [comment]
@annablender , this is ready to be merged now and you can just create v1.3.1 in GH Releases using the release notes we've reviewed here.

**Comment by Patricol** [comment]
Do we know when a release will be created?

I've been waiting on this for a few months.

Thanks for all the great work!

**Comment by snasovich** [comment]
@Patricol , unfortunately we're still blocked by an issue on Hashicorp's side per https://github.com/rancher/terraform-provider-rke/issues/346#issuecomment-1164513836. I'm going to ping them again and see if there is any update.

**Pull request merged**